src/controls/RegionDropdown.tsx

```diff
@@ -28,27 +28,85 @@ type Props = {
   region: string;
   /** Callback function for when the project ID is changed by the dropdown */
   onRegionChange: (projectId: string) => void;
+  fromSection?: string;
```

Comment:
Instead of passing bigquery down to RegionDropdown, lets create BigQueryRegionDropdown if we want to put all this bq specific logic in here.  It looks like the BQ dropdown shares very little code with RegionDropdown anyways.

Reply:
Resolved - Created BigQueryRegionDropdown for BQ specific